src/library/scala/collection/mutable/Seq.scala

```diff
@@ -42,6 +42,27 @@ trait SeqOps[A, +CC[_], +C <: AnyRef]
     */
   @throws[IndexOutOfBoundsException]
   def update(idx: Int, elem: A): Unit
+
+  /** Sorts this $coll in place according to an Ordering.
+    *
+    * @see [[scala.collection.SeqOps.sorted]]
+    * @param  ord the ordering to be used to compare elements.
+    * @return modified input $coll sorted according to the ordering `ord`.
+    */
+  def sortInPlace[B >: A]()(implicit ord: Ordering[B]): this.type
+
+  /** Sorts this $coll in place according to a comparison function.
+    *
+    * @see [[scala.collection.SeqOps.sortWith]]
+    */
+  def sortInPlaceWith()(lt: (A, A) => Boolean): this.type = sortInPlace()(Ordering.fromLessThan(lt))
```

Comment:
Another code-style bikeshed: I think the extra parentheses in sortInPlace{With,By} are unneeded since they already have a parameter list, unlike the bare sortInPlace.